### PR TITLE
Re-write the new feedback page and thanks page.

### PR DESCRIPTION
**Pull request by alexdesi**
### Jira link
https://transformuk.atlassian.net/browse/HOTT-1384

### What?
Re-write the new feedback page and a thanks page.
The new page does not use Ajax and flash messages.

### Why?
To make it Gov UK standard and improve its accessibility.

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -57,9 +57,12 @@
   get 'help', to: 'pages#help', as: 'help'
   get 'help/cn2021_cn2022', to: 'pages#cn2021_cn2022', as: 'cn2021_cn2022'
   get 'geographical_areas/:id', to: 'geographical_areas#show', as: :geographical_area
+
+  resources :feedbacks, controller: 'feedback', only: %i[new create]
+
   get 'feedback', to: 'feedback#new'
```
Comment (willfish) [inline config/routes.rb:63]: Can you remove this chipolata?

Comment (alexdesi) [reply config/routes.rb:63]: I know, it seems redundant but we need it to
use the URL `/feedback` like `/feedbacks/new`

**Comment by alexdesi** [comment]
Screenshots:
Screenshot 2022-03-14 at 15 50 48

Screenshot 2022-03-14 at 15 50 05

Screenshot 2022-03-14 at 15 49 52

**Comment by alexdesi** [comment]
Screenshot 2022-03-14 at 15 55 59

**Review by willfish — APPROVED** [review]

**Review by willfish — APPROVED** [review]
Poifect

**Pull request merged**